### PR TITLE
Adding airgap installation in README

### PR DESCRIPTION
**Pull request by shivamkm07**
# Description

This is for adding documentation about airgap dapr installation.

## Issue reference

Please reference the issue this PR will close(partially): https://github.com/dapr/docs/issues/2299

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,6 +137,28 @@ You can install Dapr runtime by pulling docker images from a given private regis
 $ dapr init --image-registry example.io/<username>
 ```
 
+#### Install in airgap environment
+
+You can install Dapr runtime in airgap(offline) environment using pre-downloaded [installer bundle](https://github.com/dapr/installer-bundle/releases). You need to download the archived bundle beforehand (e.g. daprbundle_linux_amd64.tar.gz, daprbundle_windows_amd64.zip) and unpack it. Thereafter use the `--from-dir` flag in init command pointing to the bundle location to initialize dapr.
```
Comment (shubham1172) [inline README.md:142]:
```suggestion
You can install Dapr runtime in airgap (offline) environment using a pre-downloaded [installer bundle](https://github.com/dapr/installer-bundle/releases). You need to download the archived bundle for your OS beforehand (e.g., daprbundle_linux_amd64.tar.gz,) and unpack it. Thereafter use the `--from-dir` flag in the init command to point to the bundle location to initialize dapr.
```

File: README.md [hunk 2]
```diff
@@ -137,6 +137,28 @@ You can install Dapr runtime by pulling docker images from a given private regis
...
+
+```bash
+# Initializing dapr in airgap environment
+$ dapr init --from-dir <path-to-bundle-location>
```
Comment (shubham1172) [inline README.md:146]:
```suggestion
dapr init --from-dir /path/to/bundle/location
```

File: README.md [hunk 3]
```diff
@@ -137,6 +137,28 @@ You can install Dapr runtime by pulling docker images from a given private regis
...
+```
+
+> Note: Assuming extracted bundle directory path is $HOME/daprbundle, <path-to-bundle-location> should be $HOME/daprbundle.
+
+> Note: Dapr Installer bundle just contains the placement container apart from the binaries and so `zipkin` and `redis` are not enabled by default. You can pull the images locally either from network or private registry and run as follows:
+```bash
+1. docker run --name "dapr_zipkin" --restart always -d -p 9411:9411 openzipkin/zipkin
```
Comment (shubham1172) [inline README.md:153]: Remove numbers as it's inside code block.

File: README.md [hunk 4]
```diff
@@ -137,6 +137,28 @@ You can install Dapr runtime by pulling docker images from a given private regis
...
+2. docker run --name "dapr_redis" --restart always -d -p 6379:6379 redis
+```
+
+Alternatively to the above, you can also have slim installation as well to install dapr without running any Docker containers in airgap mode.
+```bash
+$ dapr init --slim --from-dir <path-to-bundle-location>
```
Comment (shubham1172) [inline README.md:159]: Remove dollar

**Review by shubham1172 — DISMISSED** [review]

File: README.md (hunk 3 above)
Comment (shivamkm07) [reply README.md]: Done, Thanks

File: README.md (hunk 4 above)
Comment (shivamkm07) [reply README.md:159]: Dollar sign is there just to have similarity with other flag descriptions:
$ dapr init --image-registry example.io/<username>
$ dapr init --runtime-version 1.0.0
$ docker network create dapr-network
etc.

Comment (shubham1172) [reply README.md:159]: Ah, the dollar sign breaks the copy functionality on the top right. We should ideally remove it from everywhere. 
image

Comment (shivamkm07) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (shivamkm07) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (shivamkm07) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -134,7 +134,29 @@ You can install Dapr runtime by pulling docker images from a given private regis
...
+
+```bash
+# Initializing dapr in airgap environment
+dapr init --from-dir <path-to-bundle-location>
+```
+
+> Note: Assuming extracted bundle directory path is $HOME/daprbundle, <path-to-bundle-location> should be $HOME/daprbundle.
+
+> Note: Dapr Installer bundle just contains the placement container apart from the binaries and so `zipkin` and `redis` are not enabled by default. You can pull the images locally either from network or private registry and run as follows:
+```bash
```
[the remark this answers is not in the document]
Comment (shivamkm07) [reply README.md:156]: Done

File: README.md [hunk 6]
```diff
@@ -134,7 +134,29 @@ You can install Dapr runtime by pulling docker images from a given private regis
...
+dapr init --image-registry example.io/<username>
+```
+
+#### Install in airgap environment
+
+You can install Dapr runtime in airgap (offline) environment using a pre-downloaded [installer bundle](https://github.com/dapr/installer-bundle/releases). You need to download the archived bundle for your OS beforehand (e.g., daprbundle_linux_amd64.tar.gz,) and unpack it. Thereafter use the `--from-dir` flag in the init command to point to the extracted bundle location to initialize Dapr.
+
+```bash
+# Initializing dapr in airgap environment
+dapr init --from-dir <path-to-bundle-location>
```
[the remark this answers is not in the document]
Comment (shivamkm07) [reply README.md]: Updated!

**Review by mukundansundar — APPROVED** [review]
lgtm. @msfussell any comments?

File: README.md [hunk 7]
```diff
@@ -134,7 +134,35 @@ You can install Dapr runtime by pulling docker images from a given private regis
 
 ```bash
 # Example of pulling images from a private registry.
-$ dapr init --image-registry example.io/<username>
+dapr init --image-registry example.io/<username>
+```
+
+#### Install in airgap environment
+
+You can install Dapr runtime in airgap (offline) environment using a pre-downloaded [installer bundle](https://github.com/dapr/installer-bundle/releases). You need to download the archived bundle for your OS beforehand (e.g., daprbundle_linux_amd64.tar.gz,) and unpack it. Thereafter use the local Dapr CLI binary in the bundle with `--from-dir` flag in the init command to point to the extracted bundle location to initialize Dapr.
```
Comment (msfussell) [inline README.md:142]:
```suggestion
You can install Dapr runtime in airgap (offline) environment using a pre-downloaded [installer bundle](https://github.com/dapr/installer-bundle/releases). You need to download the archived bundle for your OS beforehand (for example daprbundle_linux_amd64.tar.gz for Linux,) and unpack it. Then use the local Dapr CLI binary included in the bundle with `--from-dir` flag with the `init` command to point to the extracted bundle location to initialize Dapr.
```

File: README.md [hunk 8]
```diff
@@ -134,7 +134,35 @@ You can install Dapr runtime by pulling docker images from a given private regis
...
+
+Move to the bundle directory and run the following command:
+
+```bash
+# Initializing dapr in airgap environment
+./dapr init --from-dir .
+```
+
+> For windows, use `.\dapr.exe` to point to the local Dapr CLI binary.
```
Comment (msfussell) [inline README.md:151]:
```bash
# Initializing dapr in airgap Windows environment
.\dapr.exe init --from-dir .
```

File: README.md [hunk 9]
```diff
@@ -134,7 +134,35 @@ You can install Dapr runtime by pulling docker images from a given private regis
...
+```
+
+#### Install in airgap environment
+
+You can install Dapr runtime in airgap (offline) environment using a pre-downloaded [installer bundle](https://github.com/dapr/installer-bundle/releases). You need to download the archived bundle for your OS beforehand (e.g., daprbundle_linux_amd64.tar.gz,) and unpack it. Thereafter use the local Dapr CLI binary in the bundle with `--from-dir` flag in the init command to point to the extracted bundle location to initialize Dapr.
+
+Move to the bundle directory and run the following command:
+
+```bash
+# Initializing dapr in airgap environment
```
Comment (msfussell) [inline README.md:147]:
```suggestion
# Initializing dapr in airgap Linux environment
```

File: README.md (hunk 8 above)
Comment (msfussell) [reply README.md:151]: Suggest making this a cut and paste for Windows, rather than just a comment. Makes the Windows users feel more inclusive!

Comment (msfussell) [inline README.md:151]:
```suggestion
```

File: README.md [hunk 10]
```diff
@@ -134,7 +134,35 @@ You can install Dapr runtime by pulling docker images from a given private regis
...
+./dapr init --from-dir .
+```
+
+> For windows, use `.\dapr.exe` to point to the local Dapr CLI binary.
+
+> If you are not running the above command from the bundle directory, provide the full path to bundle directory as input. For example, assuming the bundle directory path is $HOME/daprbundle, run `$HOME/daprbundle/dapr init --from-dir $HOME/daprbundle` to have the same behavior.
+
+> Note: Dapr Installer bundle just contains the placement container apart from the binaries and so `zipkin` and `redis` are not enabled by default. You can pull the images locally either from network or private registry and run as follows:
```
Comment (msfussell) [inline README.md:155]:
```suggestion
> Note: The Dapr Installer bundle just contains the Placement server container and the binaries. Zipkin and Redis containers are not included and therefore are not enabled by default. You can pull the images for these locally either from network or a private registry and run as follows:
```

File: README.md [hunk 11]
```diff
@@ -134,7 +134,35 @@ You can install Dapr runtime by pulling docker images from a given private regis
...
+
+```bash
+docker run --name "dapr_zipkin" --restart always -d -p 9411:9411 openzipkin/zipkin
+docker run --name "dapr_redis" --restart always -d -p 6379:6379 redis
+```
+
+Alternatively to the above, you can also have slim installation as well to install dapr without running any Docker containers in airgap mode.
```
Comment (msfussell) [inline README.md:162]:
```suggestion
Alternatively you can also use the `slim` installation to install Dapr without running any Docker containers.   
```

**Review by msfussell — COMMENTED** [review]
LGTM, I added some suggested wording updates

**Pull request merged**